tests/fixtures/controllers/getController.ts

```diff
@@ -78,6 +80,11 @@ export class GetTestController extends Controller {
     return {} as GetterInterfaceHerited;
   }
 
+  @Get('ModuleRedeclarationAndNamespace')
+  public async getModuleRedeclarationAndNamespace(): Promise<TsoaTest.TestModel73> {
```

Comment:
This method should make tsoa produce additional OpenAPI specification. We should test that this happens and the produced piece of OAS is sound.
https://github.com/lukeautry/tsoa/blob/master/tests/unit/swagger/schemaDetails3.spec.ts
https://github.com/lukeautry/tsoa/blob/master/tests/unit/swagger/definitionsGeneration/definitions.spec.ts

Reply:
Is this ok?